docs/how-to/llm-fine-tuning-optimization/llm-inference-frameworks.rst

```diff
@@ -28,18 +28,9 @@ graphs, tensor parallel multi-GPU, GPTQ, AWQ, and token speculation.
 Installing vLLM
 ---------------
 
-1. To install vLLM, run the following commands.
-
-   .. code-block:: shell
-
-      # Install from source
-      git clone https://github.com/ROCm/vllm.git    
-      cd vllm
-      PYTORCH_ROCM_ARCH=gfx942 python setup.py install #MI300 series
-
 .. _fine-tuning-llms-vllm-rocm-docker-image:
 
-2. Run the following commands to build a Docker image ``vllm-rocm``.
+1. Run the following commands to build a Docker image ``vllm-rocm``.
```

Comment:
we can add the following two lines before the `docker build` command:
```
git clone https://github.com/vllm-project/vllm.git
cd vllm
```

Reply:
Got this already
```      
      git clone https://github.com/vllm-project/vllm.git
      cd vllm
      docker build -f Dockerfile.rocm -t vllm-rocm .
```

Reply:
thanks. you are correct.